src/index.html

```diff
@@ -10,23 +10,40 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Document</title>
+    <title>Search bar airbnb</title>
     <link
       rel="stylesheet"
-      href="style.css"
+      href="./style.css"
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="big-searchbar"
+      action="http://localhost:8080/"
+      method="post"
+      data-qa="big"
+    >
+      <input
+        class="input input--big"
```

Comment:
Remember - the input is part of the Form Block
https://en.bem.info/methodology/quick-start/#membership

Reply:
Alright, I tried using the best of the BEM methodology. I hope I've got it right this time.